roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -150,8 +155,7 @@ spec:
         - containerPort: 8443
           protocol: TCP
         args:
-          - --tls-key-file=/certs/{{ dashboard_tls_key_file }}
```

Comment:
We still need the if else that was here for tls key and cert args vs auto generate certificates

Reply:
Also, all of the variables that allow a user to use a custom cert and key. Looks like your last commit removed that.

Reply:
ok